### ISSUE TITLE
[BUG] date conversion error

### ISSUE THREAD

**Issue by Letgamer** [GoToolSharing/htb-cli#125]
**Describe the bug**

**To Reproduce**
1. Add api key 
2. Download the binary into /usr/bin
3. Execute: htb-cli info --batch
4. See error:
```
2024-02-19T07:09:43.506-0500    ERROR   cmd/info.go:393         {"error": "date conversion error: parsing time \"%!s(<nil>)\" as \"2006-01-02 15:04:05\": cannot parse \"%!s(<nil>)\" as \"2006\""}
github.com/GoToolSharing/htb-cli/cmd.glob..func4
        /home/runner/work/htb-cli/htb-cli/cmd/info.go:393
github.com/spf13/cobra.(*Command).execute
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:876
github.com/spf13/cobra.(*Command).ExecuteC
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990
github.com/spf13/cobra.(*Command).Execute
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
github.com/GoToolSharing/htb-cli/cmd.Execute
        /home/runner/work/htb-cli/htb-cli/cmd/root.go:46
main.main
        /home/runner/work/htb-cli/htb-cli/main.go:8
runtime.main
        /opt/hostedtoolcache/go/1.20.13/x64/src/runtime/proc.go:250
```

**Comment by Letgamer** [comment]
I also tried all othe install methods, none of them are working

**Comment by QU35T-code** [comment]
Hey @Letgamer,

Sorry for delay. Duplicate with #118. Should be fixed with #127
